snow/consensus/snowman/snowmantest/block.go

```diff
@@ -0,0 +1,102 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package snowmantest
+
+import (
+	"cmp"
+	"context"
+	"time"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/snow/choices"
+	"github.com/ava-labs/avalanchego/utils"
+)
+
+const (
+	GenesisHeight        uint64 = 0
+	GenesisUnixTimestamp int64  = 1
+)
+
+var (
+	_ utils.Sortable[*Block] = (*Block)(nil)
+
+	GenesisID        = ids.GenerateTestID()
+	GenesisTimestamp = time.Unix(GenesisUnixTimestamp, 0)
+	GenesisBytes     = GenesisID[:]
+	Genesis          = BuildChain(1)[0]
```

Comment:
nit: I'd note here why we don't return a genesis block here, and we only list attributes. We may want to try and  modify tests like `TestBootstrapOldBlockAfterStateSync` in the future to return the genesis block